Update functions-bindings-register.md (#96552)
Azure/azure-content: articles/azure-functions/functions-bindings-register.md

<sup>1</sup> Version 3.x of the extension bundle currently doesn't include the [Table Storage bindings](./functions-bindings-storage-table.md). If your app requires Table Storage, you'll need to continue using the 2.x version for now.

> [!NOTE]

> > When you specify a custom version range in host.json, we recommend you use a version value from this table.

Reviewer: @rapopescu16 with all due respect, your edit changes the meaning of the sentence. What do you think about this suggestion:
```suggestion
> Even though host.json supports custom ranges for `version`, you should use a version value from this table.
```
